packages/markers/src/browser/problem/problem-composite-tree-node.spec.ts

```diff
@@ -0,0 +1,308 @@
+/********************************************************************************
+ * Copyright (C) 2021 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { enableJSDOM } from '@theia/core/lib/browser/test/jsdom';
+let disableJSDOM = enableJSDOM();
+
+import URI from '@theia/core/lib/common/uri';
+import { expect } from 'chai';
+import { Container } from '@theia/core/shared/inversify';
+import { Diagnostic, Range, DiagnosticSeverity } from '@theia/core/shared/vscode-languageserver-types';
+import { Event } from '@theia/core/lib/common/event';
+import { MarkerManager } from '../marker-manager';
+import { MarkerInfoNode, MarkerNode, MarkerOptions, MarkerRootNode } from '../marker-tree';
+import { PROBLEM_OPTIONS } from './problem-container';
+import { ProblemManager } from './problem-manager';
+import { ProblemTree } from './problem-tree-model';
+import { FileService } from '@theia/filesystem/lib/browser/file-service';
+import { ProblemCompositeTreeNode } from './problem-composite-tree-node';
+import { Marker } from '../../common/marker';
+
+disableJSDOM();
+
+let rootNode: MarkerRootNode = {
+    visible: false,
+    id: 'theia-problem-marker-widget',
+    name: 'MarkerTree',
+    kind: 'problem',
+    children: [],
+    parent: undefined
+};
```

Comment:
Since you set it already on `beforeEach`:

```suggestion
let rootNode: MarkerRootNode;
```

Reply:
I updated the pull request and followed all your suggestions. I didn´t change the “severity” attribute in the MarkerInfoNote. Should I perform one of my suggested changes or is it fine like this?